src/test/java/io/muserver/murp/ReverseProxyTest.java

```diff
@@ -49,8 +44,8 @@
 
 public class ReverseProxyTest {
 
-    private static final HttpClient client = HttpClientBuilder.httpClient()
-        .withMaxRequestHeadersSize(32768)
+    private static final HttpClient client = HttpClientUtils.createHttpClientBuilder(true)
+        .followRedirects(HttpClient.Redirect.NEVER)
```

Comment:
Maybe `ReverseProxyBuilder.createHttpClientBuilder()` should return this? (But with security enabled - or possibly allow the trust-all boolean to be passed in which would cancel my other comment about removing the trust-all code).

Reply:
added ReverseProxyBuilder.createHttpClientBuilder